tx-parser/Cargo.toml

```diff
@@ -10,3 +10,9 @@ combine = "2.1.1"
 
 [dependencies.mentat_tx]
   path = "../tx"
+
+[dependencies.mentat_parser_utils]
+  path = "../parser-utils"
+
+[dependencies.mentat_query_parser]
```

Comment:
I don't expect a dependency between parsing transactions and parsing queries.  Anything that's shared is really about parsing `edn::Value`, I expect, and should live in the parsing utilities.